lua/nvim-treesitter.lua

```diff
@@ -4,6 +4,9 @@ local info = require "nvim-treesitter.info"
 local configs = require "nvim-treesitter.configs"
 local statusline = require "nvim-treesitter.statusline"
 
+-- Ensure Neovim treats .circom files as the "circom" filetype
+vim.cmd "autocmd BufRead,BufNewFile *.circom set filetype=circom"
```

Comment:
Sorry but this requires a PR to vim first to add the filetype. Please don't add it here

Reply:
ok